app/routes/__index/domains/index.tsx

```diff
@@ -1,11 +1,32 @@
-import { Heading } from '@chakra-ui/react';
+import { Container, Heading } from '@chakra-ui/react';
+import { useNavigate } from '@remix-run/react';
+import type { DomainsTableAction } from '~/components/domains-table';
+import DomainsTable from '~/components/domains-table';
+import DOMAINS_MOCK from '~/mocks/domains';
 
 export default function DomainsIndexRoute() {
+  const navigate = useNavigate();
+
+  function onDomainAction(domainID: number, action: DomainsTableAction) {
+    switch (action) {
+      case 'EDIT':
+        navigate(domainID.toString());
+        break;
+      case 'DELETE':
+        // TODO: implement delete functionaty
+        break;
+      case 'RENEW':
+        // TODO: implement renew functionaty
+        break;
+    }
+  }
+
   return (
-    <div>
-      <Heading as="h1" size="3xl">
+    <Container maxW="contianer.xl">
+      <Heading as="h1" size="2xl" mt="8">
         Domains List
       </Heading>
```

Comment:
We might need to lower the size of this header, I don't know if it's the way the photo was taken, but it looks way too big

Reply:
Should we lower the size of the container? I just think that page looks a bit cramp (maybe that's what we should be going for? Not sure...)

Reply:
I guess using `container.xl` size for max width is optimal for page container, don't see a good reason to use a smaller one and have more whitespace

Reply:
And regarding header, I already lower the size that was before me, can reduce it even more

Reply:
he is how it looks with xl size instead of 2xl, better to me 

<img width="1588" alt="image" src="https://user-images.githubusercontent.com/32075241/218290050-44785f96-c1ed-4f0f-91f6-a7f9e455921b.png">

Reply:
`xl` looks nicer, btw are we going to add a description for this like we do on Figma?

Reply:
Yea, something like this

<img width="1588" alt="image" src="https://user-images.githubusercontent.com/32075241/218290268-84db3ff0-ae02-4c6d-9708-c1c5711e0e33.png">

Reply:
I can add it to this PR too

Reply:
We probably should ask @humphd as to what we should have on the description (if needed) so we can implement it now on your pr

Reply:
I also feel like we should drop "List" from Domains List, @Ririio what do you think?

Reply:
I agree we can go with just 'Domains' so we follow the header format